src/_testing_/CalculatorUI.test.js

```diff
@@ -0,0 +1,89 @@
+/*eslint-disable*/
```

Comment:
- [ ]  It will be great to fix the linter errors instead of disabling the linters. Remember linters are there to help you write clean code